tools/migrate/main.go

```diff
@@ -466,16 +468,6 @@ func migrate(srcDir, dstDir string, rehash bool) error {
 			Standalone: comp.Standalone,
 			Hidden:     comp.Hide,
 		}
-
-		bytes, err := cjson.Marshal(signedManifests[component.ID])
```

Comment:
Why delete this part?

Reply:
The versions are updated with `SetVersions()` on line 518, this block does not have any real affect to the final output.